src/pybito.cpp

```diff
@@ -824,8 +824,9 @@ PYBIND11_MODULE(bito, m) {
           "Add parent/child subsplit pair to DAG.")
       .def("add_nodes", &GPDAG::AddNodes)
       .def("add_edges", &GPDAG::AddEdges)
-      .def("fully_connect", &GPDAG::FullyConnect,
-           "Adds all valid edges with present nodes to the DAG.")
+      .def(
+          "fully_connect", [](GPDAG &self) { self.FullyConnect(); },
+          "Adds all valid edges with present nodes to the DAG.")
       // ** I/O
```

Comment:
Python complains with this version because it doesn't know about the return type. I've removed the return value, since we don't need it on the python side and that's less work than exposing it here.

Reply:
It looks like you made these changes.